### PR TITLE
SafeSnap: "to" param on "Raw Transaction" should be optional  fixed 557

### LINKED ISSUE #557: SafeSnap: "to" param on "Raw Transaction" should be optional
On the SafeSnap transaction builder currently when doing a "Raw Transaction" the parameter "to" is required, this make transaction like deploying a contract with SafeSnap not possible as deploying a contract don't require a "to" param, we should make this param optional instead. 

To test it, you can go here: https://snapshot.org/#/fabien.eth/create 
Click "Add transaction batch" then select type "Raw Transaction".

### PR DESCRIPTION
**Pull request by NftTopBest**
Fixes #557  .

Changes proposed in this pull request:
- make the "to" address input option
  - make the UiInput component has the prop `required` has default value `true`
  - pass {required: false} into the PluginSafeSnapInputAddress component
- but don't test it as I have no idea how to make the form submit-able yet

### CODE REVIEW

File: src/components/Ui/Input.vue [hunk 1]
```diff
@@ -21,14 +25,13 @@ function handleInput(e) {
 </script>
 
 <template>
-  <UiButton
-    class="text-left width-full mb-2 d-flex px-3"
-    :class="{ 'border-red': error }"
-  >
+  <UiButton class="text-left width-full mb-2 d-flex px-3" :class="{ 'border-red': error }">
```
Comment (bonustrack) [inline src/components/Ui/Input.vue:28]: Any idea why your linter change these?

Comment (NftTopBest) [reply src/components/Ui/Input.vue]: I add multiple extention, that maybe not correct for the project, I disable them now and use prettier seems it's the same as yours now.

Comment (samuveth) [reply src/components/Ui/Input.vue]: @NftTopBest still there are a lot of lint differences in this PR.

Comment (samuveth) [reply src/components/Ui/Input.vue]: You should be able to fix them by doing `yarn` and `yarn lint` no?

Comment (NftTopBest) [reply src/components/Ui/Input.vue]: > You should be able to fix them by doing `yarn` and `yarn lint` no?

I try yarn lint, the code result don't have change ;)

Comment (NftTopBest) [reply src/components/Ui/Input.vue]: just push a new format code, pls check
I disable other format plugin and install vetur.

Comment (NftTopBest) [reply src/components/Ui/Input.vue]: @samuveth

**Comment by samuveth** [comment]
@NftTopBest formatting still doesn't look right

**Comment by NftTopBest** [comment]
> @NftTopBest formatting still doesn't look right

pls check ;)

**Pull request closed without merging**